docs/sphinx/developer/components/pipeline/units.rst

```diff
@@ -6,7 +6,7 @@ The SimEng pipeline units provide a ``tick`` method, which performs a single cyc
 The available units are:
 
 * ``FetchUnit``: Reads instruction data from memory, to produce a stream of macro-ops.
-* ``DecodeUnit``: Reads macro-ops from the input, converts them into SimEng instruction objects as micro-ops, and writes them to the output.
+* ``DecodeUnit``: Reads macro-ops from the input, splits them into pre-decoded SimEng instruction objects stored in the macro-ops as micro-ops, and writes them to the output.
```

Comment:
I think this should be re-written to be clearer. I'm not sure exactly how, but in my opinion "splits them into pre-decoded SimEng instruction objects stored in the macro-ops as micro-ops" is not intuative to understand.